pyproject.toml

```
@@ -33,7 +33,7 @@ classifiers = [
 python = "^3.8"
 GitPython = "^3.0.0"
 colorama = "^0.4.0"
-termcolor = "^2.0.0"
+termcolor = ">= 2.0.0, < 4.0.0"
```

I recommend no upper pin.

Here's a long explanation: https://iscinumpy.dev/post/bound-version-constraints/

```suggestion
termcolor = ">= 2.0.0"
```